cmd/node/main.go

```diff
@@ -383,8 +391,30 @@ func getSuite(config *config.Config) (crypto.Suite, error) {
 
 func startNode(ctx *cli.Context, log logger.Logger, version string) error {
 	log.Trace("startNode called")
+	workingDir := getWorkingDir(ctx, log)
+
+	var err error
+
+	withLogFile := ctx.GlobalBool(logFile.Name)
+	if withLogFile {
+		var fileForLogs *os.File
+		fileForLogs, err = createLogFile(workingDir)
+		if err != nil {
+			return fmt.Errorf("%w creating a log file", err)
+		}
+
+		defer func() {
+			_ = fileForLogs.Close()
+		}()
+
+		err = logger.AddLogObserver(fileForLogs, &logger.PlainFormatter{})
```

Comment:
Might want to attach the `logger` to `fileForLogs` in the same place where `stderr` is redirected to `fileForLogs`. They're currently in separate functions (the other being `createLogFile()`).

Reply:
done